torch/_dynamo/skipfiles.py

```diff
@@ -163,6 +163,12 @@ def _module_dir(m: types.ModuleType):
 FBCODE_SKIP_DIRS = {
     "torchrec/distributed",
     "torchrec/fb/distributed",
+    # JaggedTensor and KeyedJaggedTensor are fairly badly behaved, because
+    # they tend to contain extremely long lists of integers that vary from
+    # run to run.  We are interested in tracing models that contain references
+    # to KJTs, but tracing an individual KeyedJaggedTensor is rarely
+    # profitable
+    "torchrec/sparse",
```

Comment:
Feels like it should be config driven, or easily modifiable in fbcode without changing dynamo source.

Reply:
Look, if you want to put it in configerator, I'll put it in configerator, but you are still going to have to review my config modifications

Reply:
I meant put it in _dynamo.config